modules/profile/manifests/jenkins/rosplugins.pp

```diff
@@ -264,192 +304,207 @@
+  }
+
+  ::jenkins::plugin { 'popper-api':
+    version => '1.16.0-6',
+    require => [ Jenkins::Plugin['trilead-api'] ]
   }
 
   ::jenkins::plugin { 'publish-over':
     version => '0.22',
-    require => [ Jenkins::Plugin['bouncycastle-api'], Jenkins::Plugin['command-launcher'], Jenkins::Plugin['jdk-tool'] ]
+    require => [ Jenkins::Plugin['bouncycastle-api'], Jenkins::Plugin['command-launcher'], Jenkins::Plugin['jdk-tool'], Jenkins::Plugin['trilead-api'] ]
   }
 
   ::jenkins::plugin { 'publish-over-ssh':
     version => '1.20.1',
-    require => [ Jenkins::Plugin['bouncycastle-api'], Jenkins::Plugin['command-launcher'], Jenkins::Plugin['jdk-tool'], Jenkins::Plugin['jsch'], Jenkins::Plugin['publish-over'], Jenkins::Plugin['structs'] ]
+    require => [ Jenkins::Plugin['bouncycastle-api'], Jenkins::Plugin['command-launcher'], Jenkins::Plugin['jdk-tool'], Jenkins::Plugin['jsch'], Jenkins::Plugin['publish-over'], Jenkins::Plugin['structs'], Jenkins::Plugin['trilead-api'] ]
   }
 
   ::jenkins::plugin { 'purge-build-queue-plugin':
     version => '1.0',
-    require => [ Jenkins::Plugin['ant'], Jenkins::Plugin['antisamy-markup-formatter'], Jenkins::Plugin['bouncycastle-api'], Jenkins::Plugin['command-launcher'], Jenkins::Plugin['external-monitor-job'], Jenkins::Plugin['javadoc'], Jenkins::Plugin['jdk-tool'], Jenkins::Plugin['junit'], Jenkins::Plugin['ldap'], Jenkins::Plugin['mailer'], Jenkins::Plugin['matrix-auth'], Jenkins::Plugin['matrix-project'], Jenkins::Plugin['pam-auth'], Jenkins::Plugin['windows-slaves'] ]
+    require => [ Jenkins::Plugin['ant'], Jenkins::Plugin['antisamy-markup-formatter'], Jenkins::Plugin['bouncycastle-api'], Jenkins::Plugin['command-launcher'], Jenkins::Plugin['external-monitor-job'], Jenkins::Plugin['javadoc'], Jenkins::Plugin['jdk-tool'], Jenkins::Plugin['junit'], Jenkins::Plugin['ldap'], Jenkins::Plugin['mailer'], Jenkins::Plugin['matrix-auth'], Jenkins::Plugin['matrix-project'], Jenkins::Plugin['pam-auth'], Jenkins::Plugin['trilead-api'], Jenkins::Plugin['windows-slaves'] ]
   }
 
   ::jenkins::plugin { 'run-condition':
     version => '1.2',
-    require => [ Jenkins::Plugin['bouncycastle-api'], Jenkins::Plugin['command-launcher'], Jenkins::Plugin['jdk-tool'], Jenkins::Plugin['token-macro'] ]
+    require => [ Jenkins::Plugin['bouncycastle-api'], Jenkins::Plugin['command-launcher'], Jenkins::Plugin['jdk-tool'], Jenkins::Plugin['token-macro'], Jenkins::Plugin['trilead-api'] ]
   }
 
   ::jenkins::plugin { 'scm-api':
-    version => '2.3.0',
-    require => [ Jenkins::Plugin['bouncycastle-api'], Jenkins::Plugin['command-launcher'], Jenkins::Plugin['jdk-tool'], Jenkins::Plugin['structs'] ]
+    version => '2.6.3',
+    require => [ Jenkins::Plugin['jdk-tool'], Jenkins::Plugin['structs'], Jenkins::Plugin['trilead-api'] ]
   }
 
   ::jenkins::plugin { 'script-security':
-    version => '1.48',
-    require => [ Jenkins::Plugin['jdk-tool'] ]
+    version => '1.70',
```

Comment:
```suggestion
    version => '1.71',
```
This is to match the version stated in file  modules/jenkins_files/files/var/lib/jenkins/scriptApproval.xml